.github/workflows/build.yaml

```diff
@@ -0,0 +1,105 @@
+              
+            - name: Building galasa-simbank-tests using maven
+              run: |
+                mvn -f galasa-simbank-tests/pom.xml deploy \
+                -Dgalasa.source.repo=https://development.galasa.dev/gh/maven-repo/obr \
+                -Dgalasa.central.repo=https://repo.maven.apache.org/maven2/ \
+                -Dgalasa.release.repo=file:${{ github.workspace }}/repo \
+                --batch-mode --errors --fail-at-end \
+                --settings ${{github.workspace}}/settings.xml
+
+            - name: Login to Github Container Registry
+              uses: docker/login-action@v3
+              with:
+                registry: ${{ env.REGISTRY }}
+                username: ${{ github.actor }}
+                password: ${{ secrets.GITHUB_TOKEN }}
+        
+            - name: Extract metadata for Simplatform image
+              id: metadata-simplatform
+              uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+              with:
+                  images: ${{ env.REGISTRY }}/${{ env.NAMESPACE }}/simplatform-artefacts
+        
+            - name: Build Simplatform image for development Maven registry
+              id: build-simplatform
+              uses: docker/build-push-action@v5
+              with:
+                context: .
+                file: dockerfiles/dockerfile.simplatform
+                push: true
+                tags: ${{ steps.metadata-simplatform.outputs.tags }}
+                labels: ${{ steps.metadata-simplatform.outputs.labels }}
+                build-args: |
+                 baseVersion=${{env.IMAGE_TAG}}
+                 dockerRepository=harbor.galasa.dev
+                 branch=main
+                
+            - name: Extract metadata for Simplatform image
+              id: metadata-simplatform-jar
+              uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+              with:
+                images: ${{ env.REGISTRY }}/${{ env.NAMESPACE }}/simplatform-jar-artefacts
+           
+            - name: Build Simplatform image for development Maven registry
+              id: build-simplatform-jar
+              uses: docker/build-push-action@v5
+              with:
+                context: .
+                file: dockerfiles/dockerfile.simplatform-amd64
+                push: true
+                tags: ${{ steps.metadata-simplatform-jar.outputs.tags }}
+                labels: ${{ steps.metadata-simplatform-jar.outputs.labels }}
+              
+            - name: Recycle application in ArgoCD
+              env: 
+                ARGOCD_AUTH_TOKEN: ${{ secrets.ARGOCD_TOKEN }}
+              run: |
+                docker run --env ARGOCD_AUTH_TOKEN=${{ env.ARGOCD_AUTH_TOKEN }} --rm -v ${{ github.workspace }}:/var/workspace ghcr.io/galasa-dev/argocdcli:main app actions run gh-maven-repos restart --kind Deployment --resource-name simplatform-gh --server argocd.galasa.dev

```

Comment:
The app name for simplatform will be `gh-simplatform` so please replace the `gh-maven-repos` here

Reply:
done